docs/clusters/observability/logging.md

```diff
@@ -27,8 +27,8 @@ Below are some sample CloudWatch Log Insight queries:
 
 ```text
 fields @timestamp, message
-| filter labels.apiName="<INSERT API NAME>"
-| filter labels.apiKind="RealtimeAPI"
+| filter cortex_labels.apiName="<INSERT API NAME>"
+| filter cortex_labels.apiKind="RealtimeAPI"
```

Comment:
Labels should be cortex specific, not the fluentbit name, IMO

E.g. labels.cortex.dev/apiKind="..."

Reply:
We want to namespace the metadata being added by Cortex logging infrastructure to reduce the chances of collision with user's keys. Instead of cortex_labels, I've renamed it to `cortex.labels`